src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DataFlowOperationVisitor.cs

```diff
@@ -2423,6 +2472,158 @@ IParameterSymbol GetMappedParameterForArgument(IArgumentOperation argumentOperat
+
+            // Compute the CFG for the invoked method.
+            var cfg = pointsToAnalysisResult?.ControlFlowGraph ??
+                copyAnalysisResult?.ControlFlowGraph ??
+                valueContentAnalysisResult?.ControlFlowGraph ??
+                DataFlowAnalysisContext.GetAnonymousFunctionControlFlowGraph(lambda);
+            if (cfg == null || !cfg.SupportsFlowAnalysis())
+            {
+                return;
+            }
+
+            // Ensure we are using the same control flow graphs across analyses.
+            Debug.Assert(pointsToAnalysisResult?.ControlFlowGraph == null || cfg == pointsToAnalysisResult?.ControlFlowGraph);
+            Debug.Assert(copyAnalysisResult?.ControlFlowGraph == null || cfg == copyAnalysisResult?.ControlFlowGraph);
+            Debug.Assert(valueContentAnalysisResult?.ControlFlowGraph == null || cfg == valueContentAnalysisResult?.ControlFlowGraph);
+
+            var interproceduralAnalysisData = GetInterproceduralAnalysisDataForStandaloneLambdaOrLocalFunctionAnalysis(
+                cfg, lambda.Symbol, pointsToAnalysisResult, copyAnalysisResult, valueContentAnalysisResult);
+
+            // Create analysis context for interprocedural analysis.
+            var interproceduralDataFlowAnalysisContext = DataFlowAnalysisContext.ForkForInterproceduralAnalysis(
+                lambda.Symbol, cfg, pointsToAnalysisResult, copyAnalysisResult, valueContentAnalysisResult, interproceduralAnalysisData);
+
+            // Execute interprocedural analysis and get result.
+            var analysisResult = TryGetOrComputeAnalysisResult(interproceduralDataFlowAnalysisContext);
+            if (analysisResult != null)
+            {
+                // Save the interprocedural result for the lambda operation.
+                // Note that we Update instead of invoking .Add as we may execute the analysis multiple times for fixed point computation.
+                _interproceduralResultsBuilder[lambda] = analysisResult;
+            }
+        }
+
+        private InterproceduralAnalysisData<TAnalysisData, TAnalysisContext, TAbstractAnalysisValue> GetInterproceduralAnalysisDataForStandaloneLambdaOrLocalFunctionAnalysis(
+            ControlFlowGraph cfg,
+            IMethodSymbol invokedMethod,
+            PointsToAnalysisResult? pointsToAnalysisResult,
+            CopyAnalysisResult? copyAnalysisResult,
+            ValueContentAnalysisResult? valueContentAnalysisResult)
+        {
+            var invocationInstance = (AnalysisEntityFactory.ThisOrMeInstance, ThisOrMePointsToAbstractValue);
+            var thisOrMeInstance = invocationInstance;
+            var pointsToValues = pointsToAnalysisResult?[cfg.GetEntry()].Data;
+            var copyValues = copyAnalysisResult?[cfg.GetEntry()].Data;
+            var valueContentValues = valueContentAnalysisResult?[cfg.GetEntry()].Data;
```

Comment:
Ah, all of this seems dead code, let me remove and also remove the corresponding analysis result parameters.
```suggestion
```